index.html

```diff
@@ -6,3 +6,15 @@
 <p id="box" class="mx-auto">
 
 </p>
+
+<div id="aboutdiv" class="row justify-content-between">
```

Comment:
`justify-content-around` is better than `justify-content-between` as it removes the need for the clunky `margin-left/right`s.